admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/components/assetSelector.js

```diff
@@ -36,11 +36,11 @@
                 'class' : 'upload_target hidden'
             });
             $form.before($iframe);
-            
+
             if (currentRedactor == null) {
-                $iframe.load(this.iframeOnLoad);
+                $iframe.on('load', this.iframeOnLoad);
             } else {
-                $iframe.load(this.iframeOnLoadRedactor);
+                $iframe.on('load', this.iframeOnLoadRedactor);
```

Comment:
@jetchegoyen just to doublecheck, did you test your change ?
I treied to do something like that and for me function was invoked before content is loaded. Maybe I did something wrong, so just to doublcheck

Reply:
@oleksiimiroshnyk yes, I tested it and worked fine. The function was called once the iframe got loaded. 
I just checked again and worked fine.